scripts/golangci.yml

```diff
@@ -1,69 +1,13 @@
 # yaml-language-server: $schema=https://json.schemastore.org/golangci-lint
-lintroller:
-  tier: "platinum"
```

Comment:
issue: this needs to be added back. We need to make sure that `lintroller` is compatible with the new major version of `golangci-lint`.